### DISCUSSION TITLE
Aquacal Tropical on RS485(?)

### DISCUSSION THREAD

**Discussion by subcritical71** [tagyoureit/nodejs-poolController#689] [RS485]
Hi, I'm just starting my controller build, going step by step so I don't F something up. I have already connected a SuperFlo VS, IntelliFlo VSF and IntelliChlor IC40 into the system. I also have some simple dashboards in home assistant working great. So far I'm really impressed. My next mountain to tackle, which isn't going so well, is my heater. It is an Aquacal Tropical Heater/Chiller. I don't see much mention of the Aquacal RS485. I've already tried multiple addresses, Ultratemp, MasterTemp, Max-E-Therm and none of those seem to communicate with my Nixie single body system. In the aqua cal manual for my unit it states to simply hook up the two wires to the Pentair controller and it should work (along with the installer options update in the heater controller). Has anyone got this working on an Aquacal? My controller is sending messages but not receiving any responses.

**Comment by subcritical71** [comment]
Did some further tinkering. I found an address that would at least return a response, but I don't think it means anything;


```
485 |   |   | heater | OCP | Heater[4] | 114 | 144,0,0,0,0,0,0,0,0,0
487 |   |   | heater | Heater[4] | OCP | 115 | 160,1,0,3,0,0,0,0,0,0
```

Thats all that is displayed, over and over....

**Reply by tagyoureit** [reply]
144 is the address that Nixie is trying to talk to the Ultratemp et al.  

There are 2 documents in this thread on [TFP](https://www.troublefreepool.com/threads/heatpump-aquacal-sq166-with-jandy-aqualink-rs.189828/).  Not sure if they are applicable to the Tropical but did you try to follow them (or something like it).

And just for future reference, there are a couple of ways to hook up heaters.  The classic way is with a fireman's switch which turns on/off a heater.  The heat pumps require the RS485 to be remotely controlled to allow for both heating and cooling.  Both of these use "two wires", technically.

**Reply by subcritical71** [reply]
Thanks! And yes, I did read those links. I may just end up going the route of the fireman's switch but I'm trying to avoid at the moment so I can keep cooling. I have dug around a bit more and while I do have the HP9 board on the HP, it is on v1.011. I found a few support documents (like this [one](https://www.aquacal.com/wp-content/uploads/manuals/landingpage/Content/Resources/CURRENT/AB0003.pdf)) where they seem to want Firmware 2.0 or higher. So I'll give them a call on Mon/Tues to see if that is the reason.

Also, my heater does show as being 'remotely controlled' on the display.

**Reply by rstrouse** [reply]
A 115 return from the heater is in response to the 114 command.  The 114 command above is telling the heater that is should be off.  The 115 is confirming it.  If it were calling for heat then byte index 1 would be 1 for cool this is 2.

If byte index 1 is set to 1 then the heater is commanded to heat.  If it is 2 then it is commanded to cool.  The setpoints on the heater are disabled if it hears  114.  The 3 in the 115 return indicates what the offset temperature should be.

It has been my experience that the UltraTemp protocol only uses a few bytes of the payload.  This normalizes the length of the payload to match the UltraTemp ETi hybrid heaters.

**Reply by subcritical71** [reply]
Thanks rstrouse, does this setup require external water temperature sensors to command the heater on? I haven't hooked those up yet as I'm still waiting on my building automation board.

**Reply by rstrouse** [reply]
Yes it does.  Honestly, I get why most manufacturers do not rely on the heater for thermostat control.  Often the processing power on those heaters is abysmal and used primarily to keep the sequencing for gas valves and such in order.  The protocol does not include the temperature for that heater and you cannot tap it without affecting the temperature output for the heater control board.

You will also find that the temperature of the water in the pipe also better matches with the temperature of the water in the pool as you get away from the heater.
